Add Colored Flash Example to deCONZ

D34DC3N73R opened the pull request:
## Proposed change
Update documentation with an example of colored flashing of hue bulbs via `deconz.configure` as a workaround since the `light.turn_on` service with `flash:long|short` and `color` does not work with deCONZ.



## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Additional discussion: https://community.home-assistant.io/t/update-deconz-integration-to-support-flash-with-color/246904

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

D34DC3N73R committed:
Add Colored Flash Example

Colored flashing of hue bulbs via `deconz.configure` as a workaround since the `light.turn_on` service with `flash:long|short` and `color` does not work with deCONZ

Kane610 reviewed (dismissed):
Awesome! Thanks for contributing. Looks good to me

D34DC3N73R committed:
Change Script to Automation

`field:` is not allowed in scripts, so updated to (and tested) automation

D34DC3N73R commented:
Ok so, I think the original script example had some errors. It should use `entity` instead of `entity_id` if I'm not mistaken, and `entity` should be in the first `data:` of the service call. I thought the nested `data:` lines were an error too but it turns out it's required so `field` can be included. I couldn't get the script to run correctly at first so I changed the example to an automation. 

The replacement automation example works too. But IMO, this updated script may be a better example with less extraneous detail.  

```
script:
  hue_lamp_flash:
    alias: Hue Lamp Flash
    sequence:
    - service: deconz.configure
      data:
        field: /state
        data:
          'on': true
          hue: 65535
          sat: 255
          bri: 255
          alert: breathe
        entity: light.hue_lamp
    - delay: '15'
    - service: deconz.configure
      data:
        field: /state
        data:
          'on': false
        entity: light.hue_lamp
    mode: single
```

Sorry for the runaround. Let me know if you'd like this to be submitted as another commit, or even as a separate PR.

frenck commented on source/_integrations/deconz.markdown:
```diff
@@ -267,6 +267,41 @@ automation:
 
+{% endraw %}
+
+#### Colored Flashing - RGB Philips Hue bulb using deconz.configure
```
New line after heading.

```suggestion
#### Colored Flashing - RGB Philips Hue bulb using deconz.configure

```

frenck commented on source/_integrations/deconz.markdown:
```diff
@@ -267,6 +267,41 @@ automation:
+#### Colored Flashing - RGB Philips Hue bulb using deconz.configure
+Note: Requires `on: true` to change color while the bulb is off. If `on:true` is specified, the bulb remains on after flashing is complete. The previous color is not saved.
+
+{% raw %}
```
Why is this escaped? Doesn't seem needed?

frenck replied:
Why is this marked resolved? The template is still escaped without reason. Please remove it.

D34DC3N73R replied:
If you're referring to the {% raw %} {% endraw %} tags, I just followed the format of the other published examples.

frenck replied:
Ok, but it isn't needed here, so please remove those.

D34DC3N73R replied:
Ok, removed the tags in the latest commit.

frenck commented on source/_integrations/deconz.markdown:
```diff
@@ -267,6 +267,41 @@ automation:
+        field: /state
+        data:
+          'on': false
+    mode: single
```
move this line to the top of the automation

frenck reviewed (changes requested):
This is not documenting flashing but a breathing flash effect, which is kinda Philips specific. I think we need to be clear on the difference in the description.

D34DC3N73R committed:
Resolved suggestions to Colored Flashing Example

frenck commented:
removed the review request, as my original suggestion in the review comment wasn't resolved.

D34DC3N73R commented:
I updated the Note: line to include Philips Hue by name. 
alert: breathe is what hue bridge uses with service light.turn_on and flash, so I think it makes sense to keep that description for users looking for a replacement.

D34DC3N73R committed:
Remove raw/endraw tags from Colored Flash Example

Removed raw/endraw tags and updated Note text to differentiate variables from standard copy. As well as adding a line about converting the example to work with light groups.

frenck commented:
Thanks 👍
